dragonfly_energy/__init__.py

```diff
@@ -1,2 +1,24 @@
 """dragonfly-energy library."""
+import importlib
+import pkgutil
+from honeybee.logutil import get_logger
 
+
+# load all functions that extends dragonfly core library
+import dragonfly_energy._extend_dragonfly
+
+
+logger = get_logger(__name__, filename='dragonfly-energy.log')
+
+#  find and import dragonfly_energy extensions
+#  this is a critical step to add additional functionalities to dragonfly_energy library.
+extensions = {}
+for finder, name, ispkg in pkgutil.iter_modules():
+    if not name.startswith('dragonfly_energy_') or name.count('_') > 2:
```

Comment:
would you think that there will be extensions for dragonfly_energy? I think we should only add this once there is a plugin otherwise we are adding a search through all the modules for no good reason.

Reply:
Sounds good. I wasn't sure if we'd add something down the line for district HVAC systems but, right now, I am leaning towards including things like that within dragonfly-energy. I'll take this code out.